WordPress/Classes/ViewRelated/System/Search/WPSearchControllerConfigurator.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
+#import "WPSearchController.h"
+
+extern const CGFloat SearchBarWidth;
+extern const CGFloat SearchBariPadWidth;
+extern const CGFloat SearchWrapperViewPortraitHeight;
+extern const CGFloat SearchWrapperViewLandscapeHeight;
+extern const NSTimeInterval SearchBarAnimationDuration;
+
+@interface WPSearchControllerConfigurator : NSObject
```

Comment:
Please add documentation explaining the purpose of this class... who is meant to consume this class and what for.  Also document the methods below to make it clear what they do.  Thanks!